rust/noosphere/src/lib.rs

```diff
@@ -1,7 +1,14 @@
-// pub mod builder;
+#[macro_use]
```

Comment:
Consider: Adding docs at the crate level indicating how to generate new bindings (invisible unless looking at `examples/generate_header.rs`) -- or rolling into a `build.rs` builder that generates headers automatically